### PR TITLE
Initialise `AllServices` test

### PR DESCRIPTION
**Pull request by doncem**
### JIRA link (if applicable) ###

[End-to-end tests for all services](https://tools.hmcts.net/jira/browse/BPS-1250)

### Change description ###

Set up test suite. Classification is a copy from bulk scan processor. Next PR for this topic - implement the test. Depending on the state of repo (not yet looked it over plus there are open PRs) will probably need some helper PRs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/application.conf [hunk 1]
```diff
@@ -4,3 +4,10 @@ storage-account-url=${STORAGE_ACCOUNT_URL}
 signing-key-der-base64=${SIGNING_KEY_DER_BASE64}
 proxy-host=proxyout.reform.hmcts.net
 proxy-port=8080
+
+tests {
+  all-services {
+    EXCEPTION=["publiclaw"]
+    NEW_APPLICATION=["cmc", "divorce", "finrem", "probate", "sscs"]
+  }
```
Comment (lgonczar) [inline src/functionalTest/resources/application.conf:12]: Why not just have this defined in the test class for all services?

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/AllServicesTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        .entrySet()
+        .stream()
+        .map(entry -> new SimpleEntry<>(
+            Classification.valueOf(entry.getKey()),
+            ((ConfigList) entry.getValue()).unwrapped()
+        ))
+        .collect(toMap(SimpleEntry::getKey, SimpleEntry::getValue));
+
+    @Test
+    void should_verify_all_services_are_in_the_test_config() {
```
Comment (lgonczar) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/AllServicesTest.java:34]: This test tests some internal stuff in tests themselves. It's not a functional/end-to-end test. As mentioned in the other comment, best to just define the config in this class as a map or sth.

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/AllServicesTest.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        // then
+        assertThat(actualServices).containsExactlyInAnyOrderElementsOf(expectedServices);
+    }
+
+    @ParameterizedTest
+    @MethodSource("serviceWithClassification")
+    void should_create_case_for_service(String service, Classification classification) {
+        // to be implemented
+        System.out.println("SERVICE: " + service + "; CLASSIFICATION: " + classification);
+    }
```
Comment (lgonczar) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/AllServicesTest.java:54]: Worth adding when implemented

**Comment by doncem** [comment]
conflict of interest. will re-visit with (more) desired/suggested approach

**Pull request closed without merging**